src/components/molecules/storage/FileListItem.tsx

```diff
@@ -44,6 +45,21 @@ const FileListItem = ({ file, arweave, arweaveWallet }: FileListItemProps) => {
     setArweaveTransaction(transaction);
   };
 
+  const pin = async (node: any, cid: string) => {
+    const myHeaders = new Headers();
+    myHeaders.append('Authorization', 'Basic aXBmcy1jbHVzdGVyOmZqZHNha2ZsZHNhZjk4ZndqZTczMg==');
```

Comment:
This value must *never* hit source code but *must* be stored in your local env file. Since you already pushed it, this value now is visible to the public! Secrets must go in your local .env.local file and can be used as process.env.REACT_APP_ vars. Since this is public now you *must* ask team platform to change their Basic auth password!

Reply:
What would need to happen to make this safe for production?
To the best of my knowledge, a password from .env would be compiled into the javascript target and served to untrusted browsers too.

Reply:
oh absolutely, that's a shortcoming of web2 based frontend applications in general - they usually "solve" that by hiding credentials on a centralized server and "authenticate" users that they consider trustful by some oauth concept. In a web3 world you're way more secure since you could identify a user by her signature she controls by her wallet. Now, in the ledger academy case we don't want to overcomplicate things - if you *really* were supposed to keep these credentials secret you'd store them by using a client side credential store and have the user provide them by themselves (e.g. by authenticating against another remote credential store that provides a set of secret credentials to everybody who knows its passphrase), effectively making the whole thing undemonstrateable for outsiders. Hence I'd suggest to just keep the credentials next to the app and bake them into the compiled code on build time - that still can easily be compromised by deciphering the code or tracing the network calls of a running app but that imho is the tradeoff for a "demo" app anyway - but it's never a good idea to plainly put that kind of information in plain source code.